src/ManifestStrategy/AddArrayOptionManifestStrategy.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\ManifestStrategy;
+
+use ReflectionClass;
+use SprykerSdk\Integrator\Dependency\Console\InputOutputInterface;
+use SprykerSdk\Integrator\IntegratorConfig;
+
+class AddArrayOptionManifestStrategy extends AbstractManifestStrategy
```

Comment:
It looks like this strategy works only with constants. Did we have a goal to write it in the way that allows to work with string as well? Or did you discuss with Rene that it is redundant (because I expected that this functionality should be done)?

Reply:
No, we haven't discussed this with Rene, I'll try to find out.

Reply:
Consulted with Mark, decided not to implement string support yet and add this idea to the backlog.

Reply:
But please add a docblock to that class that clarifies this.